<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>774</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>13</number>
    </property>
    <property name="topMargin">
     <number>13</number>
    </property>
    <property name="rightMargin">
     <number>13</number>
    </property>
    <property name="bottomMargin">
     <number>13</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>15</number>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>15</height>
           </size>
          </property>
          <property name="text">
           <string>Файлы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>248</width>
             <height>684</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>15</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>15</height>
           </size>
          </property>
          <property name="text">
           <string>Редактор</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="undoRedoEnabled">
           <bool>true</bool>
          </property>
          <property name="acceptRichText">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Введите Markdown текст здесь...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>15</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>15</height>
           </size>
          </property>
          <property name="text">
           <string>Предпросмотр</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="EditableWebEngineView" name="webView" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPDF"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuSet_Header">
     <property name="title">
      <string>Set Header</string>
     </property>
     <addaction name="actionHeader1"/>
     <addaction name="actionHeader2"/>
     <addaction name="actionHeader3"/>
     <addaction name="actionHeader4"/>
     <addaction name="actionHeader5"/>
     <addaction name="actionHeader6"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="menuSet_Header"/>
   </widget>
   <widget class="QMenu" name="menuAppearance">
    <property name="title">
     <string>Appearance</string>
    </property>
    <widget class="QMenu" name="menuFont_Size">
     <property name="title">
      <string>Font Size</string>
     </property>
     <addaction name="action1"/>
     <addaction name="action2"/>
     <addaction name="action3"/>
     <addaction name="action4"/>
    </widget>
    <addaction name="actionDark_Mode"/>
    <addaction name="actionLight_Mode"/>
    <addaction name="separator"/>
    <addaction name="menuFont_Size"/>
    <addaction name="separator"/>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionUnderline"/>
    <addaction name="actionSetColor"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAppearance"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>CapsLock</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDark_Mode">
   <property name="text">
    <string>Dark Mode</string>
   </property>
  </action>
  <action name="actionLight_Mode">
   <property name="text">
    <string>Light Mode</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="actionPDF">
   <property name="text">
    <string>Export to PDF</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionSetColor">
   <property name="text">
    <string>Set Color</string>
   </property>
  </action>
  <action name="actionHeader1">
   <property name="text">
    <string>Header 1</string>
   </property>
  </action>
  <action name="actionHeader2">
   <property name="text">
    <string>Header 2</string>
   </property>
  </action>
  <action name="actionHeader3">
   <property name="text">
    <string>Header 3</string>
   </property>
  </action>
  <action name="actionHeader4">
   <property name="text">
    <string>Header 4</string>
   </property>
  </action>
  <action name="actionHeader5">
   <property name="text">
    <string>Header 5</string>
   </property>
  </action>
  <action name="actionHeader6">
   <property name="text">
    <string>Header 6</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditableWebEngineView</class>
   <extends>QWidget</extends>
   <header>test</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>